app/models/project.rb

```diff
@@ -639,6 +639,9 @@ def check_status
     update_column(:status_checked_at, DateTime.current)
     return if url.blank?
 
+    # "Hidden" is a state set by admins, and we don't want to override that decision.
+    return if status == "Hidden"
```

Comment:
```suggestion
    return if hidden?
```
Could use this convenience method from the Concern: https://github.com/librariesio/libraries.io/blob/c66742fbc77f20ebdc310e887e18a95ad9b6db11/app/models/concerns/status.rb#L20-L22